cgmanifest.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Registrations": [
+        {
+            "Component": {
+                "Type": "git",
+                "Other": {
+                    "RepositoryUrl": "https://github.com/bilke/cmake-modules",
+                    "CommitHash": "66fa2579d36f30cdab93d1dbd4c3d682a25ae5f3"
```

Comment:
nit: Since you are using the latest commit from the repo (rather than the latest commit of that particular file) for libssh below, do the same here

https://github.com/bilke/cmake-modules/commits/master